migrations/20220611-add-column-alias-to-pipeline.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable new-cap */
```

Comment:
can delete this file if you're no longer adding aliasName to the pipeline model

Reply:
Yes, I have question, will it work fine if we are directly adding a new field in settings which is present in pipeline model?

Reply:
yeah should be captured in `settings` field
https://github.com/screwdriver-cd/data-schema/blob/master/migrations/20201007-add-column_settings_to_pipelines.js